label_images.py

```diff
@@ -40,7 +40,7 @@ def analyze_with_openai(image_path:str):
                 "content": [
                     {
                         "type": "text",
-                        "text": "Rate the road quality on a scale from 0 to 100. Your response should be in the form {ROAD_QUALITY: N}, where N is a number between 0 and 100. If the image does not contain a road, please enter 'NO_ROAD'. If the image is indoors, please enter 'INDOOR'."
+                        "text": "Rate the road quality on a scale from 0 to 100. Your response should be in the form {ROAD_QUALITY: N}, where N is a number between 0 and 100. If the image does not contain a road, please enter 'NO_ROAD'. If the image is indoors, please enter 'INDOOR'. Start at a score of N=100. If the pavement quality looks rough lower your score by 20. If the road contains some cracks, lower your score by 40. If the road contains a lot of cracks, lower your score by 50. If a road contains a pothole, lower your score by 80."
```

Comment:
add comma after "quality looks rough",

Will it know to not output a negative score?

Reply:
Comma added! 

As for a negative score, I haven't seen that yet. Hopefully the instructions that: "N is a number between 0 and 100" is enough to avoid that.